### PR TITLE
feat: add variable show and set statement and other docs bug fix

### LINKED ISSUE #43: add the desc about how to set enable_trace
v0.4.1 added the new system variable `enable_trace`, we need to add description about how to set enable_trace

### PR DESCRIPTION
**Pull request by jingchen2222**
1. add set statement
2. add show statement
3. fix window example
4. fix some links

### CODE REVIEW

File: reference/sql/dql/WINDOW_CLAUSE.md [hunk 1]
```diff
@@ -201,7 +201,7 @@ WINDOW w1 AS (UNION t2, t3 PARTITION BY col1 ORDER BY col5 ROWS BETWEEN 3 PRECED
 
 ```SQL
 SELECT col1, col5, sum(col2) OVER w1 as w1_col2_sum FROM t1
-WINDOW w1 AS (UNION t2 PARTITION BY col1 ORDER BY col5 ROWS BETWEEN 3 PRECEDING AND CURRENT ROW INSTANCE_NOT_IN_WINDOW) limit 10;
+WINDOW w1 AS (UNION t2 PARTITION BY col1 ORDER BY col5 ROWS BETWEEN 10s PRECEDING AND CURRENT ROW INSTANCE_NOT_IN_WINDOW);
```
Comment (lumianph) [inline reference/sql/dql/WINDOW_CLAUSE.md]: this should be ROWS_RANGE?

File: reference/sql/ddl/SET_STATEMENT.md [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+- 会话系统变量一般以`@session前缀，形如`SET @@session.execute_mode = "offline"。`注意⚠️：会话系统变量也可以选择直接以`@@`为前缀，即`SET @@execute_mode = "offline"`和前面的配置语句是等价的。变量名是大小写不敏感的。
+- OpenMLDB的SET语句只能用于设置/修改已存在（内置的）的系统变量。
+
+## 目前支持的系统变量
+
+### SESSION 系统变量
+
+| SESSION系统变量                        | 变量描述                                                     | 变量值                | 默认值    |
+| -------------------------------------- | ------------------------------------------------------------ | --------------------- | --------- |
+| @@session.execute_mode                 | OpenMDLB在当前会话下的执行模式。目前支持"offline"和"online"两种模式。<br />在离线执行模式下，只会导入/插入以及查询离线数据。<br />在在线执行模式下，只会导入/插入以及查询在线数据。 | "offline" \| "online" | "offline" |
```
Comment (dl239) [inline reference/sql/ddl/SET_STATEMENT.md]: add `@@execute_mode` example the same as `enable_trace`

File: reference/sql/ddl/SHOW_VARIABLES_STATEMENT.md [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+  enable_trace    false
+  execute_mode    online
+ --------------- --------
+```
+
+
+
+## 相关SQL语句
+
+[SET VARIABLE](../ddl/SET_VARIABLE_STATEMENT.md)
```
Comment (dl239) [inline reference/sql/ddl/SHOW_VARIABLES_STATEMENT.md]: invalid link

File: reference/sql/ddl/SET_STATEMENT.md (hunk 2 above)
Comment (jingchen2222) [reply reference/sql/ddl/SET_STATEMENT.md]: check the section of ### 配置enable_trace

**Comment by lumianph** [comment]
the README also needs to update for new sections

**Review by dl239 — APPROVED** [review]

**Review by tobegit3hub — APPROVED** [review]
LGTM

**Review by lumianph — APPROVED** [review]

**Pull request merged**